src/main/java/org/ibp/api/java/study/GermplasmStudySourceService.java

```diff
@@ -0,0 +1,16 @@
+package org.ibp.api.java.study;
+
+import org.generationcp.middleware.service.api.study.germplasm.source.GermplasmStudySourceDto;
+import org.generationcp.middleware.service.api.study.germplasm.source.GermplasmStudySourceRequest;
+
+import java.util.List;
+
+public interface GermplasmStudySourceService {
+
+	public List<GermplasmStudySourceDto> getGermplasmStudySourceList(final GermplasmStudySourceRequest germplasmStudySourceRequest);
```

Comment:
remove final

Reply:
same comment than in MIddleware, using List at the end will make it more difficult to understand, specially the count. I will prefer to use plural for the name of these functions

Reply:
Done